esp/logging/logginglib/loggingagentbase.cpp

```diff
@@ -119,11 +120,15 @@ bool CDBLogAgentBase::updateLog(IEspUpdateLogRequestWrap& req, IEspUpdateLogResp
             StringBuffer updateDBStatement;
             if(!buildUpdateLogStatement(logRequestTree, logDB.str(), table, logID, updateDBStatement))
                 throw MakeStringException(EspLoggingErrors::UpdateLogFailed, "Failed in creating SQL statement.");
+            if (getEspLogLevel()>=LogNormal)
```

Comment:
Both of them have the same function. I can change to ESPLOG to save a line of code.